### ISSUE TITLE
Evaluate and fix issue known to OpenAM as #201801-04

### ISSUE THREAD

**Issue by siepkes** [WrenSecurity/wrenam#13]
Since we share a common heritage with OpenAM the issue described [here](https://backstage.forgerock.com/knowledge/kb/article/a31060891) as "Issue #201801-04: Open Redirect" probably affects wren:AM too.

We need to evaluate if and how this issue affects wren:AM and fix it.

**Comment by pavelhoral** [comment]
The issue is with handling `goto` and `gotoOnFail` parameters. There are 2 affected components: [`RESTLoginView`](https://github.com/WrenSecurity/wrenam/blob/master/openam-ui/openam-ui-ria/src/main/js/org/forgerock/openam/ui/user/login/RESTLoginView.js) and [`CommonConfig`](https://github.com/WrenSecurity/wrensec-ui/blob/master/forgerock-ui-commons/src/main/js/config/process/CommonConfig.js) (part of *Commons UI* project).

I can see that latest commits updates to UI (maybe together with introduction of React) added special [`gotoUrl`](https://github.com/WrenSecurity/wrenam/blob/master/openam-ui/openam-ui-ria/src/main/js/org/forgerock/openam/ui/user/login/gotoUrl.jsm), but that is just a parameter wrapper without any sanitization / validation logic. 

TL;DR This issue applies to us.
